hasweb/templates/index.html

```diff
@@ -128,16 +117,181 @@
+    background-color: {{ color.primary_dark }};
+    border-color: {{ color.primary_dark }};
+    filter: brightness(85%);
+  }
+</style>
+</script>
+<script id='event-card' type='text/ractive'>
+  <div class="col-md-6 col-sm-6">
+    <div class="event-card">
+      <div class="event-card-title-block">
+        <h2 class="event-card-title">{{ title }}</h2>
+        <small class="event-card-title-date">{{ date }}, {{ city }}</small>
+      </div>
+      <p class="event-card-blurb">{{ blurb }}</p>
+      {{#if url}}
+      <a href="{{ url }}" class="btn btn-sm btn-info event-card-btn-url">Attend</a>
+      {{/if}}
+    </div>
+  </div>
+</script>
+<script id='sevent-card' type='text/ractive'>
+  <li class="col-md-6 col-sm-6">
+    <a href="{{ url }}">
+      <div class="card-title">
+        <h2>{{ title }}</h2>
+        <h3>{{ date }}, {{ venue }}, {{ city }}.</h3>
+      </div>
+      <div class="card-body">
+        {{ blurb }}
+      </div>
+    </a>
+  </li>
+</script>
 <script>
-    window.scrollback = {
-      streams:['hasgeek'], theme: 'light', ticker: true, minimize: true
-    };
-    /***** don't edit below *****/
-    (function(d,s,h,e){e=d.createElement(s);e.async=1;
-    e.src=h+'/client.min.js';scrollback.host=h;
-    d.getElementsByTagName(s)[0].parentNode.appendChild(e);}
-    (document,'script',location.protocol+'//scrollback.io'));
+
+    $(document).ready(function (argument) {
+      loadEvents();
+      loadConferences();
+    });
+
+    function loadEvents() {
+      $.getJSON("https://hasgeek.github.io/events/events.json")
+        .done(function success(data) {
+          Event = Ractive.extend({
+            template: '{{>event-card}}',
+            computed: {
+              date: function () {
+                startTime = new Date(this.get('start_time'));
+                endTime = new Date(this.get('end_time'));
+                if (startTime.toDateString() === endTime.toDateString()) { // same day 
+                  return moment(startTime).format('ddd, D MMM, hA')+"-"+moment(endTime).format("hA");
+                }
+                else { // multiple days
+                  return moment(startTime).format("D")+"–"+moment(endTime).format("D MMM YYYY");
+                }
+              }, 
+              buttonTextColor: function(color) { return idealTextColor(this.get('color.primary_dark')); }
+            }
+          });
+          ractive = new Ractive({
+            el: 'events-container',
+            template: '{{#events:id}}<event/>{{/events}}',
+            data: {
+              events: data.events
+            },
+            components: { event: Event }
+          });
+        })
+        .fail(function( jqxhr, textStatus, error ) {
+          var err = textStatus + ", " + error;
+          console.log("Request Failed: " + err );
+        });
+    }
+
+    function loadConferences() {
+      $.getJSON("https://hasgeek.github.io/events/conferences.json")
```

Comment:
I suggest moving this URL to `hascore` and further, fetching the JSON from flask instead of jQuery. That way, it will let us cache this server-side.